step.yml

```diff
@@ -36,67 +36,78 @@ source_code_url: https://github.com/bitrise-steplib/steps-brew-install
 support_url: https://github.com/bitrise-steplib/steps-brew-install/issues
 
 type_tags:
-  - installer
+- installer
+
+project_type_tags:
```

Comment:
This makes `unified-ci` skip the E2E tests on Linux stacks, but I think it also improves the workflow editor search